website/docs/using-react-context.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: using-react-context
+title: Using React Context API
+sidebar_label: Using React Context API
+---
+
+## React Context API 
+React Context API provides a easy way to pass data through the component tree without having to pass props down 
+manually at every level. You can find more about the Context API in [React documentation](https://reactjs.org/docs/context.html).
+
+You can use the Context API with React Native Navigation. In this example, we are going to create a screen which uses
+the Counter Context.
```

Comment:
> NOTE for myself, add a info:

As React Native Navigation screens are not part of the same component tree, updating the values in the shared context does not trigger a re-render across all screens. However you can still use the `React.Context` per  RNN screen component tree.